sherpa/astro/io/__init__.py

```diff
@@ -67,9 +70,29 @@
 from sherpa.io import _check_args
 from sherpa.utils import is_subclass
 from sherpa.utils.err import ArgumentErr, DataErr, IOErr
-from sherpa.utils.numeric_types import SherpaFloat
+from sherpa.utils.numeric_types import SherpaFloat, SherpaUInt
+
+from .types import KeyType, NamesType, HdrTypeArg, HdrType, DataType, \
+    Header, HeaderItem, Column, Block, TableBlock, ImageBlock, \
+    BlockList, BlockType, SpecrespBlock, MatrixBlock, EboundsBlock
+
+# Responses can often be send as the Data object or the instrument
+# version, so support it would be good to support this with types
+# like
+#
+#  RMFType = Union[DataRMF, RMF1D]
+#
```

Comment:
Every time this comes up, I wonder why DataRMF and RMF1D are different objects. I know we've looked at this a number of times, and there are some differences, but maybe at some point we can make it one - or at some point I will understand why we can't.

Reply:
My assumption is that it is likely a [SOLID](https://en.wikipedia.org/wiki/SOLID)-like design choice - the RM1D class adds extra abilities to the DataRMF so we can delegate the respobsibility to the DaraRMF class and let RMF1D focus on what it needs to do.